src/python/WMComponent/PhEDExInjector/PhEDExInjectorPoller.py

```diff
@@ -92,6 +148,33 @@ def setup(self, parameters):
 
         return
 
+    def algorithm(self, parameters):
+        """
+        _algorithm_
+
+        Poll the database for uninjected files and attempt to inject them into
+        PhEDEx.
+        """
+        logging.info("Running PhEDEx injector poller algorithm...")
+
+        self.pollCounter += 1
+
+        if self.blocksToRecover:
+            logging.info("""PhEDExInjector Recovery:
```

Comment:
I don't want to touch this PR if not absolutely needed. As that log message is exactly the same as before, can we just create another issue later ?